.github/workflows/build.yml

```diff
@@ -269,11 +270,17 @@ jobs:
             ${{ github.workspace }}/packages/serverless/dist-awslambda-layer/*.zip
 
   job_browser_integration_tests:
-    name: Browser Integration Tests
+    name: Browser Integration Tests (${{ matrix.browser }})
     needs: job_build
     runs-on: ubuntu-latest
-    timeout-minutes: 15
+    timeout-minutes: 10
```

Comment:
Reducing this from 15 to 10 (another value we use in jobs in this workflow) to emphasize these tests should run relatively fast. The 10 min timeout is for each browser/job.